ros2param/ros2param/verb/set.py

```diff
@@ -37,14 +37,29 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             'node_name', help='Name of the ROS node')
         arg.completer = NodeNameCompleter(
             include_hidden_nodes_key='include_hidden_nodes')
-        parser.add_argument(
-            '--include-hidden-nodes', action='store_true',
-            help='Consider hidden nodes as well')
+
         arg = parser.add_argument(
-            'parameter_name', help='Name of the parameter')
+            'parameters', nargs='*',
+            help='List of parameter name and value pairs i.e. "int_param 1 str_param hello_world"')
         arg.completer = ParameterNameCompleter()
+
         parser.add_argument(
-            'value', help='Value of the parameter')
+            '--include-hidden-nodes', action='store_true',
+            help='Consider hidden nodes as well')
```

Comment:
Can you move this code back above?  I think it is more logical to have all of the flags first, and then the "rest of" the arguments as that last item here.  This has no practical effect on how this operates, it is just a bit easier to read (and should also reduce the diff a bit).